README.md

```diff
@@ -939,6 +939,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Ron Swanson Quotes](https://github.com/jamesseanwright/ron-swanson-quotes#ron-swanson-quotes-api) | Television | No | Yes | Unknown |
 | [STAPI](http://stapi.co) | Information on all things Star Trek | No | No | No |
 | [SWAPI](https://swapi.co) | Star Wars Information | No | Yes | Unknown |
+| [The one API](https://the-one-api.herokuapp.com/) | The Lord of the Rings API | `apiKey` | Yes | Unknown |
```

Comment:
Your title shouldn't end with API
```suggestion
| [The one](https://the-one-api.herokuapp.com/) | The Lord of the Rings API | `apiKey` | Yes | Unknown |
```

Reply:
I see, but it's the proper name of the API, and without the "API" in the end it makes no sense. As a compromise I'd suggest renaming it this way:

```suggestion
| [The Lord of the Rings](https://the-one-api.herokuapp.com/) | The Lord of the Rings API | `apiKey` | Yes | Unknown |
```

Reply:
👌let's do this

Reply:
I'll commit :)